app/controllers/concerns/solidus_wms/additional_orders_actions.rb

```diff
@@ -0,0 +1,33 @@
+require 'axlsx'
+
+module SolidusWms
+  module AdditionalOrdersActions
+    extend ActiveSupport::Concern
+
+    def export_xlsx
+      exporter = Rails.application.config.order_xls_export_class.new
```

Comment:
Yes after studying that repo can see it just monkey patches methods onto Order and Store object instances which is not what we need here for this static config value